apm-collector/apm-collector-storage/collector-storage-define/src/main/java/org/apache/skywalking/apm/collector/storage/dao/ui/ICpuMetricUIDAO.java

```diff
@@ -20,12 +20,38 @@
 
 import java.util.List;
 import org.apache.skywalking.apm.collector.storage.base.dao.DAO;
+import org.apache.skywalking.apm.collector.storage.table.jvm.CpuMetricTable;
 import org.apache.skywalking.apm.collector.storage.ui.common.Step;
 import org.apache.skywalking.apm.collector.storage.utils.DurationPoint;
 
 /**
+ * Interface to be implemented for execute database query operation
+ * from {@link CpuMetricTable#TABLE}.
+ *
  * @author peng-yongsheng
+ * @see org.apache.skywalking.apm.collector.storage.table.jvm.CpuMetricTable
+ * @see org.apache.skywalking.apm.collector.storage.StorageModule
  */
 public interface ICpuMetricUIDAO extends DAO {
+
+    /**
+     * Execute a database query operation, implemented in different
+     * storage module.
```

Comment:
~~Execute a database query operation, implemented in different storage module.~~